### ISSUE TITLE
Problem adding via SPM

### ISSUE THREAD

**Issue by behzod-rajabov** [aws-amplify/amplify-ui-swift-liveness#182]
### Describe the bug

![Screenshot 2024-11-10 at 21 48 43](https://github.com/user-attachments/assets/1120f5a7-bca1-4a9e-a5bb-7c287dc313a3)
I am getting this problem when I add a Package to a Project via SPM. How can I fix this?

### Steps To Reproduce

```swift
Problem adding via SPM
```

### Expected behavior

Problem adding via SPM

### Swift Liveness Version

1.3.3

### Xcode version

15

### Is this a regression?

No

### OS Version

macOS 14.6.1

### Device

iPhone 15 Pro Max

**Comment by mattcreaser** [comment]
Hi @behzod-rajabov, can you please check if your git configuration is set to force signatures for annotated tags? You can check that by running `git config --get tag.forcesignannotated` - if that returns `true`, you can disable it using `git config --global tag.forcesignannotated false`.

This setting requires annotated tags to be signed with GPG, and the tags for this repository are not signed.

**Comment by behzod-rajabov** [comment]
Hi @mattcreaser This setting is returning me the "false". But the problem is still there.

**Comment by ruisebas** [comment]
@behzod-rajabov I was unable to reproduce this, so it seems like a local issue with your SPM cache. 

I'd advice you to check if either of the following fixes the issue:

1. Delete the DerivedData folder
2. On Xcode, go to **File -> Packages -> Reset Package Caches**
3. Delete your SPM cache
    ```bash
    rm -rf ~/Library/Caches/org.swift.swiftpm
    rm -rf ~/Library/org.swift.swiftpm
    ```

**Comment by behzod-rajabov** [comment]
Hi @ruisebas I have done all the options you gave, but unfortunately the problem still occurs.

**Comment by ruisebas** [comment]
In the screenshot you shared, you can see it explicitly says it's attempting to fetch a _cached_ version.

Maybe try removing the package from the list of **Recenty Used**?
![image](https://github.com/user-attachments/assets/a5039379-5fb3-4503-9c7a-20d365df0068)

**Comment by behzod-rajabov** [comment]
Yes I tried all possible methods. I tried adding it to other projects as well. But the problem is the same.

**Comment by behzod-rajabov** [comment]
@ruisebas Yes I tried all possible methods. I tried adding it to other projects as well. But the problem is the same.

**Comment by ruisebas** [comment]
I wiped everything on my end as well to be sure, but I still can't reproduce it.

Are you perhaps using SourceTree? There's [a recent bug](https://jira.atlassian.com/browse/SRCTREE-8176) that causes issues with SPM because it adds the following to the `~/.gitconfig` file:
```
[safe]
    bareRepository = explicit
```
So if you have those entries in your config file, please remove them and try again.

----
If that still doesn't work (or you don't have those entries), one last thing you could try is to run the proper `swift package` commands to clean up:

```bash
swift package purge-cache
swift package reset
```
Also make sure your Xcode is closed when you run them.

**Comment by behzod-rajabov** [comment]
Yes @ruisebas , the problem is with SourceTree. Thanks for your help 👍🏼
